### ISSUE TITLE
IKO Functional Bugs

### ISSUE THREAD

**Issue by Zerrisx** [magefree/mage#6469]
- [ ] Rielle the Everwise only draws one card when discarding two to Cathartic Reunion
![image](https://user-images.githubusercontent.com/59861365/80251666-9c32f880-863b-11ea-9d8d-a0b6d626a73c.png)

- [ ] Valient Rescuer is still freezing games (likely due to a bad Watcher)

- [ ] Kinnan, Bonder Prodigy + Jegantha the Wellspring interaction still slightly wrong - the extra mana generated by Kinnan is inheriting Jegantha's restriction when it should not

- [ ] Nyxbloom Ancient + Jegantha the Wellspring interaction also wrong - the extra mana generated by Nyxbloom is *not* inheriting Jegantha's restriction when it should

- [ ] Stormwild Capridor is missing the prevention text in chat
![image](https://user-images.githubusercontent.com/59861365/80251844-e3b98480-863b-11ea-8cd9-5dc61295f317.png)
For Comparison, Swans of Bryn Argoll:
![image](https://user-images.githubusercontent.com/59861365/80251896-f7fd8180-863b-11ea-8d38-3a75ed201cb7.png)

- [ ]  Mutual Destruction still doesn't get flash with an Ashcoat Bear in play

**Comment by ThomasArb** [comment]
Is the bug with Mutual Destruction fixed ? Because I'm not able to reproduce it.

**Comment by ThomasArb** [comment]
I fix the issue for Stormwild Capridor in the request above this comment.

**Comment by Zerrisx** [comment]
> 
> 
> Is the bug with Mutual Destruction fixed ? Because I'm not able to reproduce it.

Yep.  Notice here I can cast Lightning Bolt, but not Mutual Destruction, despite having an Ashcoat Bear in play:
![image](https://user-images.githubusercontent.com/59861365/82951814-7fa51b80-9f6d-11ea-96d7-461b4548249e.png)

**Comment by Zerrisx** [comment]
> 
> 
> I fix the issue for Stormwild Capridor in the request above this comment.

Verified fixed.
